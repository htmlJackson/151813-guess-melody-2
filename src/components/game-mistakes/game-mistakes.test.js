import React from "react";
import renderer from "react-test-renderer";

import GameMistakes from "../game-mistakes/game-mistakes.jsx";

describe(`Проверка <GameMistakes>`, () => {

  it(`Компонент <GameMistakes> корректно отрисован`, () => {
    const tree = renderer.create(<GameMistakes mistakes={2} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
