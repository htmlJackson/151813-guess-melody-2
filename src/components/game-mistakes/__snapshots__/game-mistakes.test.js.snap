// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверка <GameMistakes> Компонент <GameMistakes> корректно отрисован 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
